Dockerfile

```
@@ -0,0 +1,13 @@
+FROM node:18-alpine
+
+WORKDIR /app
+
+COPY package*.json ./
```

we only have package.json right - more than enough. 

```suggestion
COPY package.json ./
```